### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by chongjh** [ssbc/ssb-client#44]
Fails while attemping to install. Any help is appreciated!

NPM version 6.4.1
Node version 10.15.3
```
{Project Directory}\node_modules\secret-handshake\node_modules\sodium-native\d
eps\libsodium>del tmp tmp2 tmp3
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: spawn C:\Program Files (x86)\MSBuild\14.0\Bin\MSBuild.exe ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
Emitted 'error' event at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:246:12)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
npm WARN test-ssb@1.0.0 No description
npm WARN test-ssb@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sodium-native@1.10.3 install: `node-gyp-build "node preinstall.js" "node postinstall.js"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the sodium-native@1.10.3 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

**Comment by christianbundy** [comment]
Hey @chongjh, sorry to hear you're running into this as well. It's caused by this bug: https://github.com/auditdrivencrypto/secret-handshake/issues/18

cc: @dominictarr

**Comment by chongjh** [comment]
Thanks for the info and the quick response, @christianbundy! Fingers crossed this will be resolved soon.

**Comment by chongjh** [comment]
https://github.com/auditdrivencrypto/secret-handshake/commit/716f5ac231658939ddd696e8bc273d500d388796 seems to have fixed the problem. ssb-client can not be installed without errors.

**Comment by christianbundy** [comment]
Woo! Happy to hear it, cheers.

**Comment by dominictarr** [comment]
that should be fixed in secret-handshake@1.1.18
